pageserver/src/http/routes.rs

```diff
@@ -1210,6 +1210,99 @@ async fn layer_map_info_handler(
     json_response(StatusCode::OK, layer_map_info)
 }
 
+#[instrument(skip_all, fields(tenant_id, shard_id, timeline_id, layer_name))]
+async fn timeline_layer_file_scan_disposable_keys(
```

Comment:
nit: "file" is a bit redundant in names like this (we can just say "layer" rather than "layer file")